### ISSUE TITLE
[Feature] Subtask reward display consolidation

### ISSUE THREAD

**Issue by RanDumSocks** [Ayagikei/LifeUp#155]
**Is your feature request related to a problem? Please describe.**
When looking at the overview of tasks, it would be beneficial to display the total reward from the subtasks and main task, instead of just the main task reward.

**Describe the solution you'd like**
Have an option (because some may not want this forced setting) which consolidates all subtask and main task rewards onto the main task display.

Upon completion of the main task, auto-complete all subtasks and receive the remaining rewards.
Tasks with partially completed subtasks show the remaining reward available.

**Describe alternatives you've considered**
Having the correct total pre-calculated combination of all parameters on the main task. Upon completion, gives a special item which upon use (API call), subtracts the correct amount of rewards for that specific task. Obviously, a lot of work for something that is just a display variable.

**Comment by zeradawit** [comment]
I agree with your idea of displaying the reward item of subtasks on the main task overview card. However, I prefer displaying just the item's icon instead of using text. This approach maintains the card's compactness while still providing detailed information.

**Comment by RanDumSocks** [comment]
@zeradawit Oh! Maybe I should clarify a little bit. This new behaviour would look identical to how it currently displays, but the coin/exp count on the tasks will instead be calculated as the sum of the main task and sub task rewards.

I didn't propose any new UI elements, just the logic behind how already existing ones are displayed. However, an icon signifying the display values are calculated might be beneficial, but I would assume you could infer that just by seeing the presence of subtasks on the task display.

If I get some time today after work, I can take some screenshot examples if I remember.

**Comment by zeradawit** [comment]
I agree either way. However, I don't often use subtasks because of their current limitations in the main task, particularly the lack of deadlines.
